scalameter-core/src/main/scala/org/scalameter/package.scala

```diff
@@ -19,6 +19,18 @@ package object scalameter extends MeasureBuilder[Unit, Double](
   MeasureBuilder.average
 ) {
 
+  implicit def inline[T: Ordering](measurer: Measurer[T]): MeasureBuilder[Unit, T] = {
```

Comment:
It seems to me that we should add an overloaded 2-argument `withMeasurer` method that takes:
- a measurer of a different type
- an aggregator/`resultFunction` of that same type

    def withMeasurer[U](m: Measurer[U], f: Seq[Quantity[U]] => Quantity[U]): MeasureBuilder[Unit, U]

This method should live on the `MeasureBuilder` class.